shared/middleware/channels_api.rb

```diff
@@ -155,7 +155,10 @@ class ChannelsApi < Sinatra::Base
     end
     bad_request unless value.is_a? Hash
     value = value.merge('updatedAt' => Time.now)
-    value = value.merge('libraryPublishedAt' => Time.now) if value['publishLibrary']
+
+    # Set libraryPublishedAt timestamp if we are publishing a project library.
+    publish_library = value.delete('publishLibrary')
```

Comment:
What does value.delete do here? (for my own learning)

Reply:
it removes `publishLibrary` from the `value` hash, which is only superficially useful to us -- we overwrite the entire project in `PEGASUS_DB[:storage_apps]` with the `value` hash, and it felt gross to write `publishLibrary` to the db

so there wasn't any user effect, but it would start gunking up our storage_apps table